cloud-core/src/main/java/cloud/commandframework/context/CommandContext.java

```diff
@@ -558,6 +559,24 @@ public void remove(final @NonNull CloudKey<?> key) {
         return this.getOptional(key).orElseGet(defaultSupplier);
     }
 
+    /**
+     * Get a value if it exists, else compute and store the value by the supplier and return it
+     *
+     * @param key             Argument key
```

Comment:
```suggestion
     * @param key             Cloud key
```

Reply:
This follows all other methods like it (eg: getOrSupplyDefault) so i'll leave it as-is

Reply:
The context stores more than just parser results, the old methods having outdated/wrong docs doesn't mean we should copy it over to new methods, at some point when I get to fixing the nullability annotations I will also fix this on old methods